common/python/utility/hex_utils.py

```diff
@@ -41,6 +41,18 @@ def is_valid_hex_str(hex_str):
     except ValueError:
         return False
 
+def mrenclave_hex_string(enclave_metadata_file):
```

Comment:
Can we invoke this function before persisting mr_enclave value to file so that we can use the value as is?

Reply:
Just realized that this python util function needs to be called from shell script to achieve the above, which may get bit complex. 
We can use this utility from kme enclave manager during its signup flow.